horace_core/sqw/@sqw/private/cut_accumulate_data_.m

```diff
@@ -73,11 +73,22 @@
 end
 
 for iter = 1:num_chunks
+    sub_bin_starts = bin_starts(sub_bin_idxs(1, iter):sub_bin_idxs(2, iter));
+    sub_bin_ends = bin_ends(sub_bin_idxs(1, iter):sub_bin_idxs(2, iter));
+
+    num_pix_in_chunk = sum(sub_bin_ends - sub_bin_starts);
+    if num_pix_in_chunk > block_size
+        warning( ...
```

Comment:
do we want one warning per too big chunk or one warning that _these_ chunks (maybe multiple) are too big.

I'd prefer to see a single warning